### DISCUSSION TITLE
Enable fingerprint scanners automaticaly.

### DISCUSSION THREAD

**Discussion by Vince972** [linuxmint/discussions#175] [Ideas]
Hello everyone,

Is it possible to think about enabling fingerprint scanners in Linux Mint out of the box ?

Thank you.

Vincent

**Comment by eco-star** [comment]
Disclaimer: This is only an assumption.

Isn't it true that fingerprint scanning in Linux is insecure by now? I heard that a fingerprint need to be save as a picture to work out and the Kernel does not have a secure implementation yet.

Maybe I am wrong here, but if this is true then it should not be enabled :o

**Reply by anaximeno** [reply]
I did not find anything about this during my researchs, can you link any sources on this?

**Reply by eco-star** [reply]
I got it from this german article, but I have to make clear that I did not verify this specific information. And I have to be honest and say that he did not extend his statement: https://curius.de/2020/12/authentifizierung-per-fingerabdruck-mit-linux-keine-gute-idee/

**Comment by synthfi** [comment]
the fingerprint scanner on my lenovo thinkpad t440s started working without any setup from me

**Comment by anaximeno** [comment]
More info on that https://wiki.debian.org/SecurityManagement/fingerprint%20authentication
